go/client/logclient.go

```diff
@@ -308,3 +309,11 @@ func (c *LogClient) GetSTH() (sth *ct.SignedTreeHead, err error) {
 	sth.TreeHeadSignature = *ds
 	return
 }
+
+// GetSTHConsistency retrieves the consistency proof between two snapshots.
+func (c *LogClient) GetSTHConsistency(first, second uint64) ([][]byte, error) {
```

Comment:
The array of byte-array return is probably fine, but did you consider instead defining a protocol buffer  containing 'repeated bytes proof' to represent the response, and returning a pointer-to-PB instead?  That would be better aligned with the other parts of the API.